downstream/assemblies/hub/assembly-synclists.adoc

```diff
@@ -1,7 +1,7 @@
 [id="assembly-synclists"]
-= Managing Red Hat Certified collections synclists in {HubName}
+= Managing Ansible Content Collection synclists in {HubName}
```

Comment:
It seems like "Ansible Content Collection" is always capitalized, but "certified collections" or "collections" alone is not. I can't find confirmation of this outside of the doc...